test/remote/gateways/remote_ixopay_test.rb

```diff
@@ -43,23 +43,23 @@ def test_failed_purchase
     assert_equal '2003', response.error_code
   end
 
-  def test_successful_authorize_and_capture
-    omit 'Not yet implemented'
-
+  def test_successful_authorize#_and_capture
     auth = @gateway.authorize(@amount, @credit_card, @options)
     assert_success auth
+    assert_equal 'FINISHED', auth.message
+    assert_not_nil auth.params['purchase_id']
+    assert_not_nil auth.params['reference_id']
```

Comment:
I'm wondering whether these two lines could be replaced with something that tests `auth.authorization`.

Reply:
We can definitely check for presence; I couldn't find any claims in the doc about how the authorization will be constituted, so I'd like to keep the existing checks, since if they do at some point diverge that may be something we'd want to know for (maybe among other things) a 3DS2 implementation.